Firestore/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
   is IN a specified array. `whereField(_:arrayContainsAny:)` finds documents
   where a specified field is an array and contains ANY element of a specified
   array.
+- [changed] Firestore SDK now uses Nanopb rather than Objective-C Protobuf
```

Comment:
You could add a “the” between “rather than” and “Objective-C”.

Reply:
Done.